modules/osd-storage-pv-aws-create-sharedvolumes.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
+
+[id="osd-storage-pv-aws-create-sharedvolumes-console_{context}"]
+= Creating SharedVolume resources using the console
+
+You must create one SharedVolume resource per file system:access point pair in each project from which you want pods to access it. To do this using the console:
+
+.Procedure
+
+. In the OpenShift web console, create and navigate to a project.
```

Comment:
s/OpenShift/{product-title}

Reply:
Actually, this is called the OpenShift console - not the {product-title} console. These products don't have their own console - they either use OCM or the OpenShift console, depending on the action being taken.